### PR TITLE
m hole punch

### PR DESCRIPTION
**Pull request by Jerimee**
#### Summary

SUMMARY: Content "m add hole puncher, remove adj from adventure book"

#### Purpose of change

punch a hole in materials, find the chad, and then glue it back in the hole

#### Describe the solution

- add hole punch item to stationary.json
- rem "dark" from africa in adventure book

#### Additional context

The original reason ("reorg materials") for this PR was decided against.  What's left is a hole puncher  and the removal of an unnecessary adjective to avoid a problematic expression.  Hope that explains  why this tiny PR exists.

I got my git all kinds of tangled up, but then I think I got it sorted out.  **This PR should *not* make any changes at all to materials.json.**

### CODE REVIEW

File: data/json/items/tool/stationary.json [hunk 1]
```diff
@@ -13,6 +13,20 @@
     "symbol": ";",
     "color": "light_gray"
   },
+  {
+    "id": "office_holepunch",
+    "type": "GENERIC",
+    "category": "tools",
+    "name": { "str": "hole puncher", "str_pl": "hole punches" },
```
Comment (Qrox) [inline data/json/items/tool/stationary.json:20]:
```suggestion
    "name": { "str": "hole puncher" },
```

**Comment by Night-Pryanik** [comment]
Hole puncher should spawn somewhere.

File: data/json/items/tool/stationary.json [hunk 2]
```diff
@@ -13,6 +13,20 @@
     "symbol": ";",
     "color": "light_gray"
   },
+  {
+    "id": "office_holepunch",
```
Comment (ZhilkinSerg) [inline data/json/items/tool/stationary.json:17]: You need to add some spawn locations to it.

File: data/json/items/tool/stationary.json [hunk 3]
```diff
@@ -13,6 +13,20 @@
...
+    "type": "GENERIC",
+    "category": "tools",
+    "name": { "str": "hole puncher" },
+    "description": "This metal tool can help you punch a single hole in a piece of paper.  I mean, you could do more if you wanted, but you'd have to do them one at a time.  Or, if you really wanted more holes with one punch, I suppose you could fold the paper...  This is a one cylinder hole puncher.",
```
Comment (AMurkin) [inline data/json/items/tool/stationary.json:21]:
```suggestion
    "description": "This metal tool can help you punch a single hole in a piece of paper.  I mean, you could do more if you wanted, but you'd have to do them one at a time.  Or, if you really wanted more holes with one punch, I suppose you could fold the paper…  This is a one cylinder hole puncher.",
```

**Comment by Jerimee** [comment]
Man yall got a lot of feedback for a hole punch! I like it!

File: data/json/items/tool/stationary.json (hunk 2 above)
Comment (Jerimee) [reply data/json/items/tool/stationary.json:17]: I know. I will.

**Pull request merged**